components/DonationTable.tsx

```diff
@@ -6,50 +6,49 @@ import { tableContents } from '../constants/table-of-contents'
 function DonationTable() {
   return (
       <div>
```

Comment:
If you want to achieve a large table add a tailwind class on line 8, where the div has no current classes for stylind add add a width of 4/5 or something similar i.e. className="w-4/5"